### PR TITLE
[SG] Expanding categories for athletic shoes

### PR DESCRIPTION
**Pull request by ornanatan**
Add these category in sporting good vertical:
Sporting Goods > Athletics > American Football > American Football Shoes
Sporting Goods > Athletics > Basketball > Basketball Shoes
Sporting Goods > Athletics > Cricket > Cricket Shoes
Sporting Goods > Athletics > Dancing > Dancing Shoes
Sporting Goods > Athletics > Field Hockey & Lacrosse > Field Hockey & Lacrosse Shoes
Sporting Goods > Athletics > Racquetball & Squash > Racquetball & Squash Shoes
Sporting Goods > Athletics > Rugby > Rugby Shoes
Sporting Goods > Athletics > Soccer > Soccer Shoes
Sporting Goods > Athletics > Tennis > Tennis Shoes
Sporting Goods > Outdoor Recreation > Climbing > Climbing Apparel & Accessories > Climbing Shoes
Sporting Goods > Outdoor Recreation > Cycling > Cycling Apparel & Accessories >  Cycling Shoes
Sporting Goods > Outdoor Recreation > Golf > Golf Shoes
Sporting Goods > Outdoor Recreation > Outdoor Games > Badminton > Badminton Shoes

### CODE REVIEW

**Review by tylerrowsell — APPROVED** [review]

File: data/attributes.yml [hunk 1]
```diff
@@ -30209,6 +30209,22 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3418
+  name: Dance style
+  description: Identifies the specific genre or type of dance
```
Comment (ricardotejedorsanz) [inline data/attributes.yml:30214]: This should include an example for consistent style guidance:

Indicates the specific genre or types of dance, e.g. flamenco, ballet

File: data/attributes.yml [hunk 2]
```diff
@@ -30209,6 +30209,22 @@ base_attributes:
...
+  handle: dance-style
+  values:
+  - dance_style__ballet
+  - dance_style__tap dance
+  - dance_style__jazz
+  - dance_style__ballroom
+  - dance_style__hip_hop
+  - dance_style__ Salsa
+  - dance_style__ flamenco
+  - dance_style__ contemporary
```
Comment (ricardotejedorsanz) [inline data/attributes.yml:30225]: A few unnecessary spaces, missing connectors, etc. in these friendly ids. Salsa should be in lower case in the friendly id.

**Review by ricardotejedorsanz — CHANGES_REQUESTED** [review]
New categories make sense.
A few formatting errors to fix.

File: data/attributes.yml (hunk 2 above)
Comment (ornanatan) [reply data/attributes.yml:30225]: fixed

**Review by ricardotejedorsanz — APPROVED** [review]

**Pull request merged**